lib/cordial/automation_templates.rb

```diff
@@ -0,0 +1,46 @@
+module Cordial
+  # Wraps all interaction with the Automation Templates resource.
+  # @see https://api.cordial.io/docs/v1/#!/automation_templates
+  class AutomationTemplates
+    include ::HTTParty
+    extend Client
+
+    # Create a new message template.
+    #
+    # @example Usage.
+    #  Cordial::AutomationTemplates.create(
+    #    key: "promo_01_20_2018",
+    #    name: "promo_01_20_2018",
+    #    channel: "email",
+    #    classification: "promotional",
+    #    base: "hourly",
+    #    subject: "One Day Only Sale",
+    #    from: "info@example.com",
+    #    reply: "info@example.com",
+    #    description: "Promotions Team",
+    #    text: "<div>Hello World</div>"
+    #  )
+    def self.create(key:, name:, channel:, classification:, base:, subject:, from:,
```

Comment:
I've noticed that

```
subject: subject,
                          fromEmail: from,
                          replyEmail: reply,
                          fromDesc: description
```

Are inside `headers`, what do you think about to set a `headers` hash instead of every individual parameter?